cmds/volumes.go

```diff
@@ -76,20 +108,25 @@ func findPendingPVS(c *k8sclient.Client, ns string) (bool, []string) {
 		for _, item := range items {
 			status := item.Status.Phase
 			if status == "Pending" || status == "Lost" {
-				pendingClaimNames = append(pendingClaimNames, item.ObjectMeta.Name)
+				pvcName := item.ObjectMeta.Name
+				pendingClaimNames = append(pendingClaimNames, pvcName)
 			}
 		}
 		if len(pendingClaimNames) > 0 {
-			return true, pendingClaimNames
+			return true, pvcs, pendingClaimNames
 		}
 	}
-	return false, nil
+	return false, nil, nil
 }
 
 func createPV(c *k8sclient.Client, ns string, pvcNames []string, sshCommand string) (Result, error) {
 
 	for _, pvcName := range pvcNames {
-		hostPath := "/data/" + pvcName
+		paths := []string{"/data", ns, pvcName}
+		strs := []string{ns, pvcName}
+		nsPvcName := strings.Join(strs, "-")
+
+		hostPath := strings.Join(paths, "/")
```

Comment:
Also move this under `paths := ...` to keep stuff together (better readability).